### PR TITLE
Project Mongo API

### PR DESCRIPTION
**Pull request by leelalaim**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,39 +1,117 @@
...
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number,
```
Comment (sssylvasss) [inline server.js:30]: Maybe delete the once that you never use?

**Review by sssylvasss — APPROVED** [review]
Hi! 
I don't have much to add.I tried all your endpoint and all worked. It looks nice and clean like always:) 

Well done!
Ylva

File: README.md [hunk 2]
```diff
@@ -1,13 +1,5 @@
...
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://mongo-api-18.herokuapp.com/
```
Comment (KarolCislowski) [inline README.md:5]: short documentation would be nice 🤓

File: server.js [hunk 3]
```diff
@@ -1,39 +1,117 @@
...
+mongoose.Promise = Promise;
+
+const songSchema = new mongoose.Schema({
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Genre",
+  },
```
Comment (KarolCislowski) [inline server.js:20]: ❤️

File: server.js [hunk 4]
```diff
@@ -1,39 +1,117 @@
...
+      trackName: {
+        $regex: new RegExp(song_name, "i"),
+      },
+    });
+    res.json(songs);
+  } else {
+    const songs = await Song.find();
+    res.json(songs);
+  }
+});
```
Comment (KarolCislowski) [inline server.js:94]: nice filtering 👍🏻

File: server.js [hunk 5]
```diff
@@ -1,39 +1,117 @@
...
+
+app.get("/songs/:songId", async (req, res) => {
+  const { songId } = req.params;
+  const singleSong = await Song.findOne({ _id: songId });
```
Comment (KarolCislowski) [inline server.js:98]: When you are using _id field you have a nice variety of very fast functions on hand:
https://mongoosejs.com/docs/api.html#model_Model.findById

**Review by KarolCislowski — APPROVED** [review]
Great API

**Pull request closed without merging**